README.md

```diff
@@ -213,3 +279,16 @@ being maintained and conflicting with [wartremover](https://github.com/wartremov
 ### SBT Tasks
 
 No special tasks are enabled for this plugin.
+
+# Releasing of the `sbt-cake` Plugin to the Sonatype Repository
+
+In order to release version `X.Y.Z` of the `sbt-cake` plugin to the `net.cakesolutions` Sonatype repository, perform the
+following actions:
+```text
+git tag -s vX.Y.Z
+sbt publishSigned sonatypeRelease
+git push --tags
+```
+
+In order for releasing to occur, individuals should have the Cake Solutions Sonatype PGP keys setup in their local
```

Comment:
Yes, we could easily add this to CI. However, we need to generate new Sonatype accounts, keys etc. just for Jenkins.

I'll add clarifying text that indicates that this situation is temporary.

I'll also generate a follow on Jira ticket for myself to chase up Sonatype regarding how we get others added to the main account.